Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -24,6 +24,7 @@ open class ChatThreadVC: _ViewController,
     public var initialReplyId: MessageId?
 
     /// Controller for observing typing events for this thread.
+    @available(*, deprecated, message: "Events are now handled by the `eventsController`.")
```

Comment:
why do we deprecate it in this PR?

Reply:
> The ChatThreadVC.didReceiveEvent delegate was always being called twice. This was due to the fact that we had channelEventsController and eventsController assigning both delegates to the ChatThreadVC

Its in the PR description. Since it is not used anymore, it is deprecated. The delegate of this controller was conflicting with the `eventsController`